tests/build/common_include_dir.srctree

```diff
@@ -16,18 +16,17 @@ PYTHON fake_grep.py -c '#include "common/AddTraceback_impl_.*h"' c.c
 
 import sys
 from Cython.Build.Dependencies import cythonize
-import platform
 import os
 
 from distutils.core import setup
 
-# os x on Travis specifically seems to crash with nthreads>0
-osx_on_travis = (platform.system() == "Darwin" and os.getenv("TRAVIS"))
+# os x on CI specifically seems to crash with nthreads>0
+osx_on_ci = (sys.platform == "darwin" and os.getenv("CI"))
```

Comment:
```suggestion
osx_on_ci = (sys.platform == "darwin" and (os.getenv("CI") or os.getenv("TRAVIS")))
```

Reply:
Travis CI already sets `CI`, so there is no need to check for both. https://docs.travis-ci.com/user/environment-variables/#default-environment-variables